pkg/storage/unified/resource/server.go

```diff
@@ -725,6 +725,8 @@ func (s *server) sleepAfterSuccessfulWriteOperation(res responseWithErrorResult,
 		}
 	}
 
+	s.log.Debug("sleeping after successful write operation", "delay", s.artificialSuccessfulWriteDelay)
```

Comment:
Can we include details of the write operation so that it helps us to find ones we want?